lib/marqeta/transaction.rb

```diff
@@ -6,12 +6,7 @@ def self.endpoint
       'simulate/authorization'
     end
 
-    def self.since(start_time)
-      params = {
-        start_date: start_time.strftime('%Y-%m-%dT%H:%M:%S.%L%z'),
-        type: 'authorization',
-        state: 'ALL'
-      }
+    def self.index(params)
```

Comment:
I guess the intention here is the client can pass any params to the gem, and it simply passes them on to the API?

This is certainly for flexible, but I'd be cautious:

Up until now, there's been a simple, stable interface. The app passes only a timestamp to the gem. It doesn't care about the underlying params names. We're benefiting from the technique of [information hiding](https://en.wikipedia.org/wiki/Information_hiding).

If we change this to allow an arbitrary set of params to be passed, we're exposing the underlying complexity to the app. And if Marqeta decided to change names of those params, we'd have to update the app, rather than (or in addition) the gem.

So let's take a step back: What's the requirement we need to implement that led to wanting this change?

Reply:
Yep I think you're right. I'll change it.